includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -204,17 +204,18 @@ public function getUserNewUpvotes( $userId ) {
 			return [];
 		}
 
-		$upvotes = $this->getUserUpvotes( $userId, $status['last_notified'] );
-
-		return $upvotes;
+		return $this->getUserUpvotes(
+			$userId,
+			!empty( $status['last_notified'] ) ? $status['last_notified'] : null
+		);
 	}
 
 	private function getUserUpvotesStatus( $userId ) {
 		$db = $this->getDatabaseForRead();
 
 		$status = ( new \WikiaSQL() )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$status". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)